src/apis/term.cpp

```diff
@@ -449,6 +449,45 @@ static int term_getPixels(lua_State* L) {
     return 1;
 }
 
+static int term_fillPixels(lua_State* L) {
```

Comment:
Could you move this into `term.drawPixels(x, y, color, width, height)` instead? Basically, just add an `if (lua_isnumber(L, 3)` around the code here, and move it into `term_drawPixels`.